packages/ai/core/prompt/standardize-prompt.ts

```diff
@@ -110,7 +103,10 @@ export function standardizePrompt<TOOLS extends ToolSet>({
     if (!validationResult.success) {
       throw new InvalidPromptError({
         prompt,
-        message: 'messages must be an array of CoreMessage or UIMessage',
+        message: [
+          'messages must be an array of CoreMessage or UIMessage',
```

Comment:
maybe change msg to "message must be a CoreMessage or a UI message" and consider changing the InvalidPromptError so that the messages can be included and printed instead